test/suites/zombie_tanssi_collator_peers/test_zombie_tanssi_collator_peers.ts

```diff
@@ -0,0 +1,582 @@
+                const deltaWidth = Math.max(...rows.map((r) => r.delta), 0).toString().length; // width for delta seconds
+
+                const lines = rows.map(
+                    ({ n, author, delta }) =>
+                        `#${n.toString().padStart(numWidth)} (+${delta.toString().padStart(deltaWidth)}s): ${author}`
+                );
+
+                console.log(lines.join("\n"));
+                const entries = [...byAuthor.entries()].sort((a, b) => b[1] - a[1]);
+                const summaryLines = entries.map(([a, c]) => `  - ${a}: ${c}`).join("\n") || "  (none)";
+                console.log(`[Container-2000] Authors summary:\n${summaryLines}`);
+
+                // -------- Run peers analysis + assertions --------
+                for (const { name, path } of collatorLogs) {
+                    const { peers, times } = await collectSeries(path);
+
+                    expect(peers.length, `${name}: no 'Idle (N peers)' lines found in ${path}`).to.be.greaterThan(0);
+
+                    const st = stats(peers);
+                    const ones = peers.filter((n) => n === 1).length;
+                    const zeros = peers.filter((n) => n === 0).length;
+                    const frac1 = ones / peers.length;
+                    const frac0 = zeros / peers.length;
+
+                    const firstTs = times[0] || "(unknown start)";
+                    const lastTs = times[times.length - 1] || "(unknown end)";
+
+                    console.log(`\n[${name}] peers over time (${peers.length} samples)`);
+                    console.log(`[${name}] window: ${firstTs}  →  ${lastTs}`);
+                    console.log(
+                        `[${name}] min=${st.min}, max=${st.max}, mean=${st.mean.toFixed(2)}, p50=${st.p50}, p90=${st.p90} | 1-peer=${(
+                            frac1 * 100
+                        ).toFixed(
+                            1
+                        )}% (${ones}/${peers.length}) | 0-peer=${(frac0 * 100).toFixed(1)}% (${zeros}/${peers.length})`
+                    );
+                    console.log(`[${name}] sparkline:\n${sparkline(peers)}\n`);
+
+                    expect(
+                        frac1,
+                        `${name}: majority of samples report exactly 1 peer (${ones}/${peers.length} = ${(frac1 * 100).toFixed(1)}%). Expected ≤ 50%.`
+                    ).to.be.at.most(0.5);
+                }
+
+                // Optional cross-check: current RPC health peers for Container2000 (non-fatal)
+                try {
+                    const health: any = await container2000Api.rpc.system.health();
+                    const peersNow =
+                        typeof health?.peers?.toNumber === "function"
+                            ? health.peers.toNumber()
+                            : (health?.toJSON?.().peers ?? 0);
+                    console.log(`[Container2000 RPC] current peers=${peersNow}`);
+                } catch {
```

Comment:
I would add at least `consol.error` here

Reply:
Removed this code